test/lib/check_notification_test.rb

```diff
@@ -15,14 +15,11 @@ class CheckNotificationTest < ActiveSupport::TestCase
   end
 
   test "should notify airbrake and not call Pusher when send notification with undefined info" do
-    Airbrake.stubs(:configured?).returns(true)
-    Airbrake.stubs(:notify).returns('notified airbrake')
+    CheckSentry.stubs(:notify).returns('notified airbrake')
```

Comment:
I know this doesn't have any impact, but just for consistency, can we replace airbrake by Sentry? :)

Reply:
Definitely! I just found a few other places where we still reference airbrake by name - going to change those, too, just to reduce confusion.